app/_src/mesh/index.md

```diff
@@ -64,7 +64,8 @@ provides a full stack connectivity platform for all of our L4-L7 connectivity,
 for both edge and internal API traffic.
 
 <center>
-  <img src="/assets/images/docs/mesh/gw_mesh.png" width="600px"/>
+  <img src="/assets/images/docs/mesh/gw_mesh.png" width="600px" 
+  alt="Diagram showing how {{site.base_gateway}} and {{site.mesh_product_name}} work together to run two products on two data centers. {{site.base_gateway}} is used to manage communication between the client and the products in each data center, but also between two different products. {{site.mesh_product_name}} is used to allow different services within a product to communicate with one another."/>
```

Comment:
@lena-larionova @fabianrbz Just wondered, will liquid variables work in alt text or should we write out the name (and then probably turn off Vale for these img elements on the page)?

Reply:
Yep, they work! You can check by inspecting the output here: https://deploy-preview-7652--kongdocs.netlify.app/mesh/latest/
Right-click the caption and go to "Inspect", you'll see both the caption and the alt text.